src/main/docker/gitlab/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM gitlab/gitlab-ce:latest
+
```

Comment:
You may want to add the modifications regarding the password reset "bug" mentioned in the documentation?